euphony/src/main/java/co/euphony/rx/AudioRecorder.java

```diff
@@ -66,8 +73,12 @@ public void start() {
 	 */
 	public void stop() {
 		if (running) {
-			audioRecord.stop();
-			running = false;
+			try {
+				audioRecord.stop();
+				running = false;
+			} catch (IllegalStateException e) {
+				Log.e(LOG, "Before calling listen(), you should acquire RECORD_AUDIO permission.");
+			}
```

Comment:
I think if `start()` is not executed, `stop()` will not be called, so wouldn't this exception handling be unnecessary?

Reply:
@yeonns Hmm, You're right.
Logically, `stop()` will not be called, beacuse of `running` status. Hawk Eye..! 👀